### ISSUE TITLE
Release: Version 14

### ISSUE THREAD

**Issue by strues** [ctrlplusb/react-universally#524]
I haven't seen any check lists going around or a discussion on releasing the next branch so here we are.

- [ ] Merge master bug fix pending PRs
- [ ] Merge master changed into Next
- [ ] Test Next
- [x] RenderToNodeStream?

What else am I forgetting? We should get this up to date and out since the release of React 16 seems to be smooth.

**Comment by oyeanuj** [comment]
Yes! We should also go through the _ideas/suggestions for 'next'_ ticket and see which of those can be included: https://github.com/ctrlplusb/react-universally/issues/428

**Comment by ctrlplusb** [comment]
Hey all! Thanks for the activity here.

Please would all contributors get active on the gitter channel. Thanks 👍

I'd like us to start using a git rebase strategy for `next` and all pull requests in order to keep the merge noise down.  If you are unfamiliar with git rebasing then please let me know and I would be happy to take you through it. 👍

Please keep all changes targetting `next` rather than `master`. I know we have already let a few slip into `master` but moving forward it would be best to use `next` to stage up some changes before me merge in.

I'll review #428 and give some more of my ideas.

Also, has anyone tried `react-loadable` against this project?  James has added some cool APIs such as the `preloadAll`.  It may be worth trying a branch out and seeing how it flies.

I'd also like to start killing the feature branches.  Really keep them down to 2 or 3 at an absolute maximum.  I know we had an issue for this already.  I'll revisit it soon.

**Comment by codepunkt** [comment]
How does this work with css-in-js libraries, e.g. styled-components? Will this be a problem for branches?

**Comment by sergiokopplin** [comment]
@ctrlplusb which branches you think will be good to maintain?

**Comment by sergiokopplin** [comment]
@codepunkt you mean 'RenderToNodeStream' with css-in-js?

**Comment by codepunkt** [comment]
@sergiokopplin Yeah. Just noticed i've commented into the wrong issue. Sorry for that!

**Comment by oyeanuj** [comment]
Another topic for v14:

https://github.com/ctrlplusb/react-universally/blob/83d533a9c780716d18f034f7fb52dbd3a1c4051b/internal/webpack/configFactory.js#L379-L383

I noticed in the configFactory here that we create client bundles in ES5. I was researching a little bit more and it seems like if we set up two bundles - one for modern browsers and one for non-modern browsers, we can make a significant dent on bundle sizes.

Still new to this topic but most of my understanding comes from this article: https://philipwalton.com/articles/deploying-es2015-code-in-production-today/

#514 and #531 seem to also be related to this.

**Comment by birkir** [comment]
I've been busy in React Native world, please let me know if you guys need help.
It's my favourite project so I'm looking forward to see in what direction we're going.

**Comment by saniko** [comment]
Hello,
Any updates on next version?

Best

**Comment by birkir** [comment]
`next` now has both webpack 4 and react 16

https://github.com/ctrlplusb/react-universally/compare/master...next

Question @ctrlplusb , are we using prettier-eslint or no? There seems to be an inconsistency between branches regarding prettier. All pull-requests are now including prettier junk, making it very hard to glance PR's.

Thanks

**Comment by ctrlplusb** [comment]
I am keen on vanilla prettier (i.e. without the prettier-eslint). We should also use eslint-preset-prettier alongside eslint-preset-airbnb in order to disable all the style rules. 👍

**Comment by oyeanuj** [comment]
@birkir Thoughts on https://github.com/ctrlplusb/react-universally/issues/524#issuecomment-340304381 with webpack 4?
